### PR TITLE
docs: standardize terminology to DataHub Cloud

### PR DESCRIPTION
**Pull request by yoonhyejin**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

Changing these (including all of the lowercase  variants) to "DataHub Cloud" 

- Acryl DataHub
- Acryl Cloud
- Managed Datahub
- Acryl instance (to Datahub Cloud instance)
- (Some of) SaaS

let me know if I miss anything 

## Summary by CodeRabbit

- **New Features**
  - Updated documentation to reflect the rebranding from "Acryl DataHub" to "DataHub Cloud" across various components, enhancing clarity and consistency.

- **Bug Fixes**
  - No bug fixes were applied in this release.

- **Documentation**
  - Revised references throughout the documentation to align with the new product name "DataHub Cloud," ensuring clarity on service offerings and features.

- **Chores**
  - Minor updates to textual content across multiple files to support the rebranding effort, without altering functionality.

### CODE REVIEW

File: datahub-web-react/src/app/home/AcrylDemoBanner.tsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ export default function AcrylDemoBanner() {
                     >
                         Schedule a demo
                     </StyledLink>{' '}
-                    of Acryl DataHub to see the advanced features that take it to the next level or purchase Acryl Cloud
+                    of DataHub Cloud to see the advanced features that take it to the next level or purchase DataHub Cloud
```
Comment (jayacryl) [inline datahub-web-react/src/app/home/AcrylDemoBanner.tsx:55]: I wonder if we need to say "or purchase DataHub Cloud"

File: docs/api/tutorials/operations.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ import TabItem from '@theme/TabItem';
 
 The Operations APIs allow you to report operational changes that were made to a given Dataset or Table using the 'Operation' concept.
 These operations may be viewed on the Dataset Profile (e.g. as last modified time), accessed via the DataHub GraphQL API, or 
-used to as inputs to Acryl Cloud [Freshness Assertions](/docs/managed-datahub/observe/freshness-assertions.md). 
+used to as inputs to DataHub Cloud [Freshness Assertions](/docs/managed-datahub/observe/freshness-assertions.md).
```
[the remark this answers is not in the document]
Comment (jayacryl) [reply docs/api/tutorials/operations.md:10]: +1

File: docs/api/tutorials/structured-properties.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ Additionally, you need to have the following tools installed according to the me
 <Tabs>
 <TabItem value="CLI" label="CLI" default>
 
-Install the relevant CLI version. Forms are available as of CLI version `0.13.1`. The corresponding SaaS release version is `v0.2.16.5`
+Install the relevant CLI version. Forms are available as of CLI version `0.13.1`. The corresponding DataHub Cloud release version is `v0.2.16.5`
```
Comment (jayacryl) [inline docs/api/tutorials/structured-properties.md:35]: nice catch!

**Review by jayacryl — APPROVED** [review]

File: datahub-web-react/src/app/home/AcrylDemoBanner.tsx (hunk 1 above)
Comment (yoonhyejin) [reply datahub-web-react/src/app/home/AcrylDemoBanner.tsx:55]: hmm make sense - but I would prefer not to include the change in this PR, maybe we should make another one

File: docs/api/tutorials/operations.md (hunk 2 above)
Comment (yoonhyejin) [inline docs/api/tutorials/operations.md:10]:
```suggestion
used as inputs to DataHub Cloud [Freshness Assertions](/docs/managed-datahub/observe/freshness-assertions.md). 
```

**Pull request merged**